README.md

```diff
@@ -1,29 +1,66 @@
 # Setup
 
-## CLI
-If you have grabbed this template from the Shopify CLI you should be good to go!
+## From Shopify CLI
+Shopify CLI will automatically set up the simulator.
 
-Start developing your awesome extension in the `./src` folder.
+Start developing your awesome extension.
 
+## From this repo
 
-## From github
-If you grabbed this template directly from github:
+### Install the CLI
+
+Download and install the `argo-admin-cli`
 
 #### npm
 ```bash
-## With npm
-npm install
-
-npm run generate -- --type=SUBSCRIPTION_MANAGEMENT
+npm install -g @shopify/argo-admin-cli
+```
 
-## With yarn
-yarn
+#### yarn
+```bash
+yarn global add @shopify/argo-admin-cli
+```
 
-yarn generate --type=SUBSCRIPTION_MANAGEMENT
+### Initialize the project
+```bash
+argo-admin-cli init --type=EXTENSION
 ```
 
+Run through the prompt, then go into your project directory
+
 ### Available extensions:
  - SUBSCRIPTION_MANAGEMENT
 
-**Note:**
-Once you're setup, you may delete the `scripts` folder as you will no longer need it.
+```bash
+cd path/to/project
+```
+
+# Local development
+
+Run a local server.
+
+#### npm
+```bash
+npm run server
+```
+
+#### yarn
+```bash
+yarn server
+```
+
+The server is on `localhost:PORT` where the default port is 39351.
+
+# Build
+
+#### npm
+```bash
+npm run build
+```
+
+#### yarn
+```bash
+yarn build
```

Comment:
Why do they need to run this? Shouldn't this be run as part of the release process?

Reply:
If you mean rails CLI, then yeah it will. I think all of our commands will be hooked up automatically. These are instructions for the off chance they only use this repo without rails CLI. Maybe for each section I should mention it'll be covered in the Shopify CLI process.

Reply:
@alanthai I think the build part wouldn't even be part of CLI. When we do a release we will already have the built files as part of the package on NPM

Reply:
Added a new section `#### CLI` in addition to npm and yarn to cover in case you didn't run `init`